sellercenter-setting/src/main/java/com/gfg/sellercenter/setting/entity/Setting.java

```diff
@@ -0,0 +1,17 @@
+package com.gfg.sellercenter.setting.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+@EqualsAndHashCode
+public class Setting {
+
+    private int sellerId;
+
+    private String label;
```

Comment:
NotNull, please

Reply:
Why not null? I have only one constructor where you have to pass all attributes so it will be never null.

Reply:
Because that will allow validation to pass when you create this object through reflection